src/Framework/Framework/Compilation/Parser/Binding/Parser/ArrayInitializerExpression.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+
+namespace DotVVM.Framework.Compilation.Parser.Binding.Parser
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class ArrayInitializerExpression : BindingParserNode
+    {
+        public IList<BindingParserNode> ElementInitializers { get; private set; }
```

Comment:
```suggestion
        public List<BindingParserNode> ElementInitializers { get; private set; }
```

I'd prefer to use List or array. The IList interface is fairly broken - it contains the Add method, but arrays can be assigned to it, so it may fail randomly at runtime...

Reply:
Other nodes use List so List it is, also the privet set has no reason, removed.